test/functional/ui/inccommand_spec.lua

```diff
@@ -727,11 +727,11 @@ describe(":substitute, inccommand=split", function()
 
     feed("x")
     screen:expect([[
-      xo lines                      |
       Inc substitution on           |
       xo lines                      |
```

Comment:
Reason for this change: Because the cursor started on the last line (after `insert()`ing the text in `common_setup()`), the temporary split (preview) bumps up the top window.